### PR TITLE
Feature/repairshopr 3759

### LINKED ISSUE #3759: RepairShopr
### App
RepairShopr

### Triggers
* New Customer
* New Invoice
* New Payment
* New Ticket
* Ticket Resolved
* Ticket Status Changed

### Actions
* Create Customer
* Create Lead
* Create Ticket
* Log Call
* Update Ticket
* Find Customer
* Find Ticket

### PR DESCRIPTION
**Pull request by vellames**
Closes #3759 

Only main sources and actions implemented

### CODE REVIEW

File: components/repairshopr/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "repairshopr"
+    ],
+    "homepage": "https://pipedream.com/apps/repairshopr",
+    "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+    "license": "MIT",
+    "publishConfig": {
+      "access": "public"
+    }
+  }
+
```
Comment (feyzullah) [inline components/repairshopr/package.json:17]:
```suggestion

```
oops there were spaces here

File: components/repairshopr/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "keywords": [
+      "pipedream",
+      "repairshopr"
+    ],
+    "homepage": "https://pipedream.com/apps/repairshopr",
+    "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+    "license": "MIT",
+    "publishConfig": {
+      "access": "public"
+    }
```
Comment (feyzullah) [inline components/repairshopr/package.json:15]: Does not this app have any dependency?

File: components/repairshopr/sources/common/source.mjs [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import app from "../../repairshopr.app.mjs";
+
+export default {
+  dedupe: "unique",
+  props: {
+    app,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
```
Comment (feyzullah) [inline components/repairshopr/sources/common/source.mjs:10]: Can we just use `DEFAULT_POLLING_SOURCE_TIMER_INTERVAL` from `@pipedream/platform` like it is done [here](https://github.com/PipedreamHQ/pipedream/blob/f8ff2ffb27265ef154c1f69c08440d70f9bc4fec/components/podio/sources/common/common-poll.mjs#L11)

**Review by feyzullah — CHANGES_REQUESTED** [review]
Hi @vellames, I guess Superdocu's `package.json` has been deleted accidentally. Can you check? And I have added just a few comments. Can you please check them too?

**Comment by vellames** [comment]
Hey @feyzullah. I mess this branch. Please continue your review by looking at #4939 please. I already applied the changes

**Pull request closed without merging**